src/view/film-card-view.js

```diff
@@ -37,15 +27,35 @@ const createFilmCardTemplate = ({
 
   const controlsItemButton = (
     `<button class="film-card__controls-item ${classesWatchlist}"
-        type="button">Add to watchlist</button>
-        <button class="film-card__controls-item ${classesWatched}"
-        type="button">Mark as watched</button>
-        <button class="film-card__controls-item ${classesFavorite}"
-        type="button">Mark as favorite</button>`
+    type="button">
+    Add to watchlist
+    </button>
+    <button class="film-card__controls-item ${classesWatched}"
+    type="button"
+    >Mark as watched
+    </button>
+    <button class="film-card__controls-item ${classesFavorite}"
+    type="button">
+    Mark as favorite
+    </button>`
```

Comment:
```suggestion
    type="button">
      Add to watchlist
    </button>
    <button 
      class="film-card__controls-item ${classesWatched}"
      type="button"
    >
      Mark as watched
    </button>
    <button 
      class="film-card__controls-item ${classesFavorite}"
      type="button"
    >
      Mark as favorite
    </button>`
```

все хорошо, но не читабельно же)) Не понятно, где начинается, где заканчивается тег.